.changeset/unlucky-poets-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": patch
```

Comment:
```suggestion
"@razorpay/blade": major
```

Reply:
Fixed in https://github.com/razorpay/blade/pull/738/commits/d93b8c3b40060a0a822ec3b74458efcc27220f16 😸